wiki/mapping/intermediate-mapping.md

```diff
@@ -4,15 +4,201 @@ prev: ./basic-mapping.md
+
+## Walls
+Walls are a versatile tool in maps that are often overlooked. They provide the most direct way to influence full body motion, as well as an immersive visual element. Overuse/misuse can lead to visual clutter or full vision blocking- so care must be taken. Treat any wall placed in either of the middle two lanes as a vision block for that lane, as they block the same (if not more) space. Additionally, walls placed in both middle lanes at the same time are never acceptable - most players cannot safely move out of the way within their playspace. Even players who are able to dodge these walls will not enjoy the amount of movement required.
+
+### Moving the Player with Walls
+Walls are a way to encourage the player to move left, right, down, or a combination of the three while playing. They are extremely powerful at doing this since players cannot touch them for too long.
+
+* Left and right movement should feel natural and never be forced
+    * The note pattern should already encourage the player to move in that direction.
+    * There should be enough space between walls if using left/right back to back.
+      * Often 1 beat between the end of one and the start of next is enough. Though depending on the BPM more space is required.
+    * Groove Walls are a situational exception to this but, are hard to execute correctly.
+* Down movement (crouch walls) is trickier as it is easier to do more harm than good
+    * Most players dislike crouch walls, be very selective when using them
+      * Crouch walls should be avoided when the spawn distance is very short and NJS is high as they make even fully telegraphed crouch walls too fast to react to.
+    * Keep them short, there’s rarely a good reason to have them longer than 1-2 beats
+    * If you are ending a crouch wall with notes just after, be extremely careful with what direction you start (up/down)
+      * Starting down too quickly is uncomfortable and almost always unnecessary. 
+      * Starting up too late is also awkward
+    * It is easy to justify a crouch wall if the lyrics in a song say “down”, “under”, or any other reference to a downward motion
+    * Side notes are more natural to hit while under a crouch wall than up/down notes
+    * It is better to have a crouch wall end early than late as it plays better for any player in general. Especially those who have physical trouble crouching for too long. 
+        * If your crouch wall is longer than 1 beat, shorten it by 0.5
+        * If less than 1 beat and greater than 0.5, shorten by 0.25
+        * Otherwise, leave it
```

Comment:
I think the list formatting here makes this section pretty hard to read.
I think instead of having just two at the highest parent level, I think it'd be better to break these apart into two separate lists with these as the titles.
So `Left and right movement should feel natural and never be forced` would be the header for the first list
and `Down movement (crouch walls) is trickier as it is easier to do more harm than good` would be the header for the 2nd list